config/processors/syslog_log_audit_tenable.nessus_scanner.conf

```diff
@@ -75,9 +76,25 @@ filter {
       dissect {
         tag_on_failure => "_dissectfailure_2"
         mapping => {
-          "rule.description" => "%{rule.description}: %{[network.network_range][gte]}-%{[network.network_range][lte]}"
+          "rule.description" => "%{?data}: %{tmp_ip}"
+        }
+      }
+      if [tmp_ip] =~ "((?:(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.){3}(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)),|-" {
```

Comment:
what is this for?

Reply:
To match ip pattern. we are trying to parse if the pattern is either "ip,ip" or "ip-ip" and map it to network.network_range. anything other those patterns (like multiple ip.s seperated by comma ',') will be taken to "rule.description"

Reply:
okay. Since ip-ip is not a valid network range. You may need to write ruby code to convert to a CIDR.